### PR TITLE
Mlops 376 add plotting examples

### PR DESCRIPTION
**Pull request by RufusGladiuz**

### CODE REVIEW

**Review by dianacarvalho1 — COMMENTED** [review]
and maybe the path could be instead of `docs/readme_documents/NP_classifier_plot.png`  just `docs/images/NP_classifier_plot.png` or something simpler

File: README.md [hunk 1]
```diff
@@ -90,6 +90,54 @@ result = client.match_spectra_from_path(
...
+```python
+from omigami import MoleculePlotter
+plotter = MoleculePlotter()
+plotter.plot_molecule_structure_grid(best_matches, 
+                                     draw_indices=True, 
+                                     molecule_image_size=[600, 600], 
+                                     substructure_highlight="C(=O)")
+```
+
+
```
Comment (dianacarvalho1) [inline README.md:107]: This is duplicated

File: README.md [hunk 2]
```diff
@@ -90,6 +90,54 @@ result = client.match_spectra_from_path(
...
+Want to plot the results of the [Classyfire](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-016-0174-y) model. The following code will handle that for you.
+```python
+from omigami import MoleculePlotter
+plotter = MoleculePlotter()
+plotter.plot_classyfire_result(best_matches)
+```
+<img src="docs/images/classyfire_plot.png" width="500">
+
+Furthermore, omigami provides the possibility to use the [NPClassifier](https://www.researchgate.net/publication/344008670_NPClassifier_A_Deep_Neural_Network-Based_Structural_Classification_Tool_for_Natural_Products) API.
```
Comment (dianacarvalho1) [inline README.md:132]:
```suggestion
Furthermore, Omigami provides the possibility to use the [NPClassifier](https://www.researchgate.net/publication/344008670_NPClassifier_A_Deep_Neural_Network-Based_Structural_Classification_Tool_for_Natural_Products) API.
```

File: README.md [hunk 3]
```diff
@@ -90,6 +90,54 @@ result = client.match_spectra_from_path(
...
+
+```python
+from omigami import MoleculePlotter
+plotter = MoleculePlotter()
+plotter.plot_NPclassifier_result(result[1], color='orange')
```
Comment (dianacarvalho1) [inline README.md:137]:
```suggestion
plotter.plot_NPclassifier_result(best_matches, color='orange')
```

File: README.md [hunk 4]
```diff
@@ -90,6 +90,54 @@ result = client.match_spectra_from_path(
...
+plotter = MoleculePlotter()
+plotter.plot_molecule_structure_grid(best_matches, 
+                                     draw_indices=True, 
+                                     molecule_image_size=[600, 600], 
+                                     substructure_highlight="C(=O)")
+```
+<img src="docs/images/molecule_plot.png " width="500">
+
+
+Want to plot the results of the [Classyfire](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-016-0174-y) model. The following code will handle that for you.
```
Comment (dianacarvalho1) [inline README.md:124]: I still don't like this sentence. It seems you are trying to sell something and it is not the same "tone" as the rest of the documentation

**Review by dianacarvalho1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (tamaralipows) [reply README.md:124]: Maybe just say "The following code allows us to plot the results of the ..."

**Review by tamaralipows — CHANGES_REQUESTED** [review]
Hey Rouven, almost there, let's just address Diana's latest comments and then we're good to go :)

File: README.md [hunk 5]
```diff
@@ -90,6 +90,37 @@ result = client.match_spectra_from_path(
 #### Notebooks
 You can find a [tutorial](https://github.com/omigami/omigami/blob/master/notebooks/ms2deepscore/tutorial.ipynb) notebook in the `/notebooks/` folder.
 
+#### Plotting
+Plotting graphs works the same way for both Spec2Vec and MA2DeepScore.
```
Comment (dianacarvalho1) [inline README.md:94]:
```suggestion
Plotting graphs works the same way for both Spec2Vec and MS2DeepScore.
```

**Review by dianacarvalho1 — CHANGES_REQUESTED** [review]

**Review by dianacarvalho1 — APPROVED** [review]
👍🏼

**Pull request merged**